rpi-imager-haos.json

```diff
@@ -0,0 +1,26 @@
+{
+    "os_list": [
+        {
+            "name": "Home Assistant OS 5.13 (Pi 4 - 64-bit)",
+            "description": "Distributon optimized to run Home Assistant Core and Add-Ons.",
+            "url": "https://github.com/home-assistant/operating-system/releases/download/5.13/hassos_rpi4-64-5.13.img.xz",
+            "icon": "https://raw.githubusercontent.com/home-assistant/version/master/rpi-imager-haos.png",
```

Comment:
Instead of this url, can we refer to `https://version.home-assistant.io/rpi-imager-haos.png` (assuming it will be deployed there after build)

Also, can we make our big logo smaller? The one with the multiple tree nodes.